### ISSUE TITLE
Issues completing the crossword using just keyboard

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1391]
If you attempt to complete the crossword without using the mouse it can be frustrating as things do not work consistently. This may not be a complete list but issues include:

- Tabbing will select and highlight words in the puzzle in order but sometimes will miss out words. This seems to happen if an across word and a down word start in the same square.
- The focus should move to the next letter in the word as you type. Sometimes this does not work. It seems to not work when you move between down words and across words (the down words will move focus as expected but the across ones will not, even if they have previously worked).
- Sometimes the focus highlight is highlighting squares a different word to the one being types. For example, an across word and down word starting in same square - the highlight shows the down word but the focus moves across as you type.
- Shift-tab to move backwards will never let you leave the crossword. You seem to always be able to get out using the tab key but using shift-tab you end up looping the crossword continually.
- Sometimes the arrow keys do not move you to the expected square. You can always get where you want eventually but it sometimes takes some going forwards and back to get there.
